src/Config.py

```
@@ -34,14 +35,46 @@ def __init__(self, configPath: str) -> None:
                             "password": accs[account]["password"]
                         }                    
                 if not self.accounts:
-                    raise InvalidCredentialsException                    
+                    raise InvalidCredentialsException
+                                    
+                if config.get("notificationAll", False):
+                    self.notificationOnStart = True
+                    self.notificationOn2FA = True
+                    self.notificationOnDrop = True
+                    self.notificationOnFault = True
+                else:
+                    self.notificationOnStart = config.get("notificationOnStart", False)
+                    self.notificationOn2FA = config.get("notificationOn2FA", False)
+                    self.notificationOnDrop = config.get("notificationOnDrop", False)
+                    self.notificationOnFault = config.get("notificationOnFault", False)
+                
+                if config.get("soundAll", False):
+                    self.soundOnStart = True
+                    self.soundOn2FA = True
+                    self.soundOnDrop = True 
+                    self.soundOnFault = True  
+                else:
+                    self.soundOnStart = config.get("soundOnStart", False)
+                    self.soundOn2FA = config.get("soundOn2FA", False)
+                    self.soundOnDrop = config.get("soundOnDrop", False)
+                    self.soundOnFault = config.get("soundOnFault", False)
+                
+                self.soundPath = config.get("soundPath","./assets/defaultNotificationSound.wav")        
+                if (self.soundOnStart or self.soundOn2FA or self.soundOnDrop or self.soundOnFault) and not Path(self.soundPath).exists():
+                    raise AssetNotFoundException(self.soundPath)
```

Is it sensible to crash instead of reporting error to Logs and proceed without sounds?

---

> Is it sensible to crash instead of reporting error to Logs and proceed without sounds?

@LeagueOfPoro suggested it to be implemented that way.